src/app/api/user/[id]/unsafemetadata/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextResponse, NextRequest } from "next/server";
```

Comment:
i don't think we need a backend route to get unsafe metadata since we can get it directly from the `useUser` hook in the frontend. this will reduce our potential failure points since instead of using our backend as a middleman (client -> backend -> clerk) we can access clerk directly (client -> clerk).

we may need to store public metadata that defines the user's admin roles. in that case, we will need a backend route to act as the setter since we don't want to allow the frontend to update admin roles. I'll make a task for this